charts/gha-runner-scale-set-controller/crds/actions.github.com_ephemeralrunnersets.yaml

```diff
@@ -47,7 +47,7 @@ spec:
                     githubServerTLS:
                       properties:
                         certConfigMapRef:
-                          description: Required
+                          description: Required, the summed size of the values of the configmap must be less than 1048576 bytes as these will be mounted as an environment variable on the listener pod.
```

Comment:
```suggestion
                          description: Required, the summed size of the values of the configmap must be less than 1048576 bytes (~1 MB) as these will be mounted as an environment variable on the listener pod.
```

Reply:
Makes it easier to understand at a glance

Reply:
Updated at the source and regenerated manifests to propagate the change 👍